### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by FlanFlanagan**
This fixes a content transfer in the DnD circles. The (package) on circles was being passed and caused the regions to take null regionNodes. Same with institutions.

### CODE REVIEW

**Comment by gonuke** [comment]
Some of these may compete with #63 .... looking now

**Comment by gonuke** [comment]
Or maybe with #64, that is already merged...

**Comment by FlanFlanagan** [comment]
They do. The drag was not working properly with what was there. The text was just reverting to the fixed location given by 

-				text.setLayoutX(getLayoutX()+getRadiusX()*0.2);		
-				text.setLayoutY(getLayoutY()+getRadiusY()*0.2);	

So in order to accommodate the change you made that provide location for text I had to adjust the dragging.

**Comment by FlanFlanagan** [comment]
If #64  doesn't match up that might just be careless merging on my part.

File: cyclist/src/edu/utexas/cycic/RegionView.java [hunk 1]
```diff
@@ -67,7 +66,9 @@ public void handle(MouseEvent event){
 				}
 			}
 		});		
-		
+		for(int i = 0; i < workingRegion.institutions.size(); i++){
```
Comment (gonuke) [inline cyclist/src/edu/utexas/cycic/RegionView.java]: is this redundant with line 47-49 or so?

Comment (gonuke) [inline cyclist/src/edu/utexas/cycic/RegionView.java]: this is the part that caught my eye

Comment (gonuke) [inline cyclist/src/edu/utexas/cycic/RegionView.java]: I think this list may get filled twice now...

**Comment by FlanFlanagan** [comment]
I think what happened is your fix to the list was included with my fix, and I never removed mine when you pushed up yours. They are redundant. I will remove mine and push up again.

**Comment by FlanFlanagan** [comment]
Done

**Comment by FlanFlanagan** [comment]
Looks like it's still okay.

**Pull request closed without merging**